.github/workflows/test.yml

```
@@ -0,0 +1,47 @@
+name: Test
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
+    services:
+      postgres:
+        image: postgres:9.5
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: digitalmarketplace_test
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
```

We should make this a string, YAML sometimes treats numbers with colons weirdly... https://docs.docker.com/compose/compose-file/#ports

```suggestion
          - "5432:5432"
```